### ISSUE TITLE
width of "window.withProgress()" is very small since the change of notification

### ISSUE THREAD

**Issue by yaegassy** [neoclide/coc.nvim#3814]
## Result from CocInfo

```
## versions

vim version: NVIM v0.8.0-dev+190-gde5ccf234
node version: v16.14.2
coc.nvim version: 0.0.80-046e00c 2022-05-12 21:53:26 +0800
coc.nvim directory: /Users/yaegassy/.local/share/nvim/plugged/coc.nvim
term: iTerm.app
platform: darwin
```

## Describe the bug

My extension called coc-intelephense (`@yaegassy/coc-intelephense`) uses `window.withProgress()`. <https://github.com/yaegassy/coc-intelephense/blob/master/src/index.ts#L331>

The width of the progress window has become very small since this "notification" change... <https://github.com/neoclide/coc.nvim/commit/7deabf3ea455b289a8f0d0e4372c8f41baf1acc6>

It appears that `notification.maxWidth` is not respected. Of course, `notification.minProgressWidth` is removed, so that is not respected either.

As a side note, the `notification.*` and `coc.preferences.enableMessageDialog` settings have not been changed. They are default values.

## Reproduce the bug

See DEMO.

### Latest

https://user-images.githubusercontent.com/188642/168123680-b93e8bf8-413d-40e8-94c9-3fcd068701b2.mp4

### Before "Notification" change

<https://github.com/neoclide/coc.nvim/commit/740ac1a3bc7336b5e6609d33211c943c9d970986>

https://user-images.githubusercontent.com/188642/168123719-6147b102-9d0a-4c7b-a17f-5e2de8fa4c28.mp4

**Comment by chemzqm** [comment]
Fixed by d103f694

**Comment by yaegassy** [comment]
Thanks for the fix. Fixes checked.

https://user-images.githubusercontent.com/188642/168220912-79e37f0f-cb0b-453b-9769-b2441a749ec4.mp4
